subsys/bluetooth/host/conn.c

```diff
@@ -3355,6 +3355,75 @@ int bt_conn_le_read_all_remote_features(struct bt_conn *conn, uint8_t pages_requ
 }
 #endif /* CONFIG_BT_LE_EXTENDED_FEAT_SET */
 
+#if defined(CONFIG_BT_FRAME_SPACE_UPDATE)
+void notify_frame_space_update_complete(struct bt_conn *conn,
+					struct bt_conn_le_frame_space_updated *params)
+{
+	struct bt_conn_cb *callback;
+
+	SYS_SLIST_FOR_EACH_CONTAINER(&conn_cbs, callback, _node) {
```

Comment:
There is a conflict with https://github.com/zephyrproject-rtos/zephyr/pull/93703 here. Whichever is merged last will need to rebase and update this.

Reply:
This has been merged now